dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/StreamJsonParser.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Json.Nodes;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.Connectors.GoogleVertexAI.Core;
+
+/// <summary>
+/// Internal class for parsing a stream of jsons into separate json objects.
```

Comment:
nit: "stream of jsons into separate json objects" -> "stream of text which contains a series of discrete JSON strings into en enumerable containing each seperate JSON string"